src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -1325,7 +1326,14 @@ void Transformations::Snippets(void) {
     }
 
     CPU_DEBUG_CAP_TRANSFORMATION_SCOPE(this, Snippets);
+// Disable MainSnippets for int8 models on arm platforms
+#if defined(OPENVINO_ARCH_ARM64)
+    if (!ov::pass::low_precision::LowPrecision::isFunctionQuantized(model, {ov::pass::low_precision::levels::int8})) {
```

Comment:
Should be same list of levels as defined in Transformations::UpToLpt

Reply:
Fixed